Better Config & Docs

ThunbergOlle opened the pull request:
* index.js : Fixed success message when you're signed in(small), Fixed so instead of the "add message" set inside the script it now reads from the config file. Fixed "gamesPlayed(:45)" so that it reads from config file.

+ install.bat: On windows you can install by just opening "install.bat" instead of having to locate the file and then type in the command.

+ start.bat: Added so you can start the script on windows with just 1 click.

* credentials.example.js: Added things that you have to config. (This will make it easier for other people to use the script and understand it)

* README.md: Better docs for installing the scripts and setting it up

ThunbergOlle committed:
Better Config & Docs

* index.js : Fixed success message when you're signed in(small), Fixed so instead of the "add message" set inside the script it now reads from the config file. Fixed "gamesPlayed(:45)" so that it reads from config file.

+ install.bat: On windows you can install by just opening "install.bat" instead of having to locate the file and then type in the command.

+ start.bat: Added so you can start the script on windows with just 1 click.

* credentials.example.js: Added things that you have to config. (This will make it easier for other people to use the script and understand it)

* README.md: Better docs for installing the scripts and setting it up

harastaivan commented:
Hello Olle,

thank you very much for your contribution. I agree with you on everything you've improved besides 1 thing:

- the `.bat` files - I don't think it's necessary to make 2 files for running basically 1 command
- on Windows people can press `Shift+Right_Click` and open console in the directory they're in if finding the directory is problem for them

Please, make another commit which reverts the changes connected to the `.bat` _thing_ and push to this PR.

Thanks.

ThunbergOlle committed:
Final fix

- install.bat

- start.bat

* Fixed README.md to the old file. Added information about the requirements of the node version.

ThunbergOlle commented:
Hi,
Updated it. I also added a requirement for the node version. (The script doesn't work because of the module "signale" with versions lower than 8 of nodejs)

👍

harastaivan commented:
Awesome! Thank you.
